src/test/java/tests/Tests.java

```diff
@@ -0,0 +1,214 @@
+package tests;
+
+import dto.ItemPair;
+import exceptions.CommonTestException;
+import org.apache.commons.collections4.CollectionUtils;
+import org.junit.jupiter.api.*;
+import parser.JsonParser;
+import parser.NoSuchFileException;
+import shop.Cart;
+import shop.RealItem;
+import shop.VirtualItem;
+import utils.Utils;
+
+import java.io.File;
+import java.io.IOException;
+import java.lang.reflect.Field;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static constants.Constants.FILE_NAME_LIST;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class Tests {
+
+
+    private static JsonParser parser = new JsonParser();
```

Comment:
what for they are static?

Reply:
Because parser is used in static method setUp (annotation @BeforeAll)